x-pack/plugins/siem_rules/lib/fetch.js

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import axios from 'axios';
+
+export const fetch = axios.create({
+  headers: {
+    Accept: 'application/json',
+    'Content-Type': 'application/json',
+    'kbn-xsrf': 'professionally-crafted-string-of-text',
+  }
+});
```

Comment:
You can change this over to use the native fetch if you want to:
https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch